hummingbot/connector/connector/balancer/balancer_connector.py

```diff
@@ -492,7 +540,7 @@ async def _api_request(self,
             err_msg = ""
             if "error" in parsed_response:
                 err_msg = f" Message: {parsed_response['error']}"
-            raise IOError(f"Error fetching data from {url}. HTTP status is {response.status}.{err_msg}")
+                raise IOError(f"Error fetching data from {url}. HTTP status is {response.status}.{err_msg}")
```

Comment:
It'd be safer to always raise exception when the response is not 200 regardless.